### PR TITLE
Feat(Reveal Lockbox xPubs)

### PR DESCRIPTION
**Pull request by schnogz**
## Description
- Blocks users from adding Lockbox XLM to their wallet if they are not using a support Lockbox browser (Chrome)
- Adds new modal to display the xPubs/addresses of all the coins available for Lockbox
- Updates wallet xPub modal to be styled similar to Lockbox

## Change Type
- Feature
- Bug Fix

## Testing Steps
- Lockbox 

## Code Checklist
- [ ] Code compiles successfully (verified via `yarn start`)
- [ ] No lint issues exist (verified via `yarn lint`)
- [ ] New and existing unit tests pass (verified via `yarn test`)
- [ ] `README.md` and other documentation is updated as needed

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/modals/Lockbox/ShowLockboxXPubs/template.js [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+              id='modals.lockbox.showxpubs.warning'
+              defaultMessage="Don't share your Extended Public Keys (xPubs) with an untrusted source. Anyone with access to these can keep track of your payments and may be able to disrupt access to your wallet."
+            />
+          </Text>
+        </WarningBanner>
+        <Tabs>
+          {keys(coins).map(coin => {
+            return (
+              <Tab
+                className={activeTab === coin ? 'active' : ''}
```
Comment (tony-blockchain) [inline packages/blockchain-wallet-v4-frontend/src/modals/Lockbox/ShowLockboxXPubs/template.js]: no keys, coin will be nice key I think

File: packages/blockchain-wallet-v4-frontend/src/modals/Lockbox/ShowLockboxXPubs/template.js [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+            defaultMessage='Close'
+          />
+        </Button>
+      </ModalFooter>
+    </Modal>
+  )
+}
+
+PromptLockbox.propTypes = {
+  coin: PropTypes.string
```
Comment (tony-blockchain) [inline packages/blockchain-wallet-v4-frontend/src/modals/Lockbox/ShowLockboxXPubs/template.js]: looks like propTypes are different

File: packages/blockchain-wallet-v4-frontend/src/modals/Wallet/ShowXPub/index.js [hunk 3]
```diff
@@ -1,19 +1,11 @@
...
-import ShowXPubTemplate from './template'
 
-class ShowXPubContainer extends Component {
-  constructor (props) {
-    super(props)
-    this.state = { step: 0 }
-  }
+import ShowXPub from './template'
 
+class ShowXPubContainer extends Component {
```
Comment (tony-blockchain) [inline packages/blockchain-wallet-v4-frontend/src/modals/Wallet/ShowXPub/index.js]: could we make it 1 file given container does nothing?

File: packages/blockchain-wallet-v4/src/redux/kvStore/lockbox/selectors.js [hunk 4]
```diff
@@ -98,6 +98,12 @@ export const getLockboxBchContext = state => {
   })
 }
 
+export const getLockboxBchXpub = (state, deviceIndex) =>
+  getDevice(state, deviceIndex)
+    .map(path(['bch', 'accounts']))
```
Comment (tony-blockchain) [inline packages/blockchain-wallet-v4/src/redux/kvStore/lockbox/selectors.js]: could be `path(['bch', 'accounts', 0, 'xpub'])`

File: packages/blockchain-wallet-v4-frontend/src/modals/Lockbox/ShowLockboxXPubs/template.js (hunk 1 above)
Comment (schnogz) [reply packages/blockchain-wallet-v4-frontend/src/modals/Lockbox/ShowLockboxXPubs/template.js]: coins is an object `{ btc: 'xpub', bch: 'xpub' ...`. Thus I'm using `keys` to get the coin name. Is there another way todo this?

**Comment by schnogz** [comment]
Test coverage added

**Pull request merged**